ZelBack/src/services/fluxService.js

```diff
@@ -701,14 +909,14 @@ async function adjustKadenaAccount(req, res) {
       const kadenaURI = `kadena:${account}?chainid=${chainid}`;
       const fluxDirPath = path.join(__dirname, '../../../config/userconfig.js');
       const dataToWrite = `module.exports = {
-  initial: {
-    ipaddress: '${userconfig.initial.ipaddress || '127.0.0.1'}',
-    zelid: '${userconfig.initial.zelid || config.fluxTeamZelId}',
-    kadena: '${kadenaURI}',
-    testnet: ${userconfig.initial.testnet || false},
-    apiport: ${Number(userconfig.initial.apiport || config.apiport)},
-  }
-}`;
+        initial: {
```

Comment:
this indentation was actually intentional. As the spaces would be written to our config file and it would look weird.

Reply:
Updated.